packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js

```diff
@@ -1,15 +1,22 @@
+const { FeatureFlags } = require('amplify-cli-core');
+const latestTranslateRegions = FeatureFlags.getNumber('latestRegionSupport.translate');
```

Comment:
for existing projects this value would be undefined correct? Just make sure that the ternary operations below are not impacted because of that :D

Reply:
Shouldn't it pick up the `defaultValueForExistingProjects` value?